xcode/Subconscious/Shared/Components/Common/Transclude/TranscludeButtonStyle.swift

```diff
@@ -12,19 +12,40 @@ struct TranscludeButtonStyle: ButtonStyle {
         cornerRadius: AppTheme.cornerRadiusLg
     )
     @Environment(\.isEnabled) private var isEnabled
-
+    @Environment(\.colorScheme) var colorScheme
+    
+    func background(_ configuration: Configuration) -> Color {
+        switch (colorScheme) {
+        case .dark:
+            return configuration.isPressed
+                ? Color.secondaryBackground
+                : Color.secondaryBackground.opacity(0.5)
+        case .light:
+            return configuration.isPressed
+                ? Color.backgroundPressed
+                : Color.clear
+        default:
+            return Color.clear
+        }
+    }
+    
+    var stroke: Color {
+        colorScheme == .light
+            ? Color.separator
+            : Color.clear
+    }
```

Comment:
Vary the transclude style based on light vs dark mode.

Reply:
Nice. I think we want to consider the wash style for light mode as well.

Perhaps tertiary background?

Reply:
Maddeningly, `tertiaryBackground` is `white` on light mode, making it useless... Here's `tertiarySystemBackgroundGrouped`, which looks ok on light but is way too bright for dark mode, imo:

<img src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/89029e60-8cb1-46c0-9dd6-338e6d8f0fd8" alt="Simulator Screenshot - iPhone 14 - 2023-07-18 at 17 23 58" width="320"> <img src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/558e575d-c4af-4682-be2c-99e31a1a4963" alt="simulator_screenshot_FAE42048-77C3-4617-900A-F01C599A2A05" width="320">

--- 

Here's my best attempt at a nice looking solution: using `tertiarySystemGroupedBackground` for light and `secondarySystemBackground` for dark.

<img src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/d11e3207-9dfb-4a28-9e50-fd8a58d3c206" alt="Simulator Screenshot - iPhone 14 - 2023-07-18 at 17 29 47" width="320"> <img src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/5243053b-f255-44c8-8c67-6fcb6f09b267" alt="Simulator Screenshot - iPhone 14 - 2023-07-18 at 17 29 57" width="320">